src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,23 @@
+server:
+  port: 8000
+
+spring:
+  application:
+    name: api-gateway
+
+eureka:
+  client:
+    service-url:
+      defaultZone: http://localhost:8761/eureka
```

Comment:
Change the `localhost` to the `registry-service`.

Reply:
It does't work if change it to "http://registry-service:8761/eureka".
Or i don't understand what you want.

Reply:
Then need to add spring profiles.
For cloud profile:
```
eureka:
  client:
    service-url:
      defaultZone: http://localhost:8761/eureka
```

Add for develop profile:
```
eureka:
  client:
    service-url:
      defaultZone: http://registry-service:8761/eureka
```

Reply:
On the cloud it should look like this:
```
http://{service-name}:{service-port}/eureka
```